charts/kafka-operator/templates/crds.yaml

```diff
@@ -20099,10 +20099,18 @@ spec:
           status:
             description: KafkaTopicStatus defines the observed state of KafkaTopic
             properties:
+              managedBy:
+                description: 'ManagedBy describes who is the manager of the Kafka
+                  topic. When its value is not ''koperator'' then modifications to
+                  the KafkaTopic CR''s topic configurations will be unaffected on
```

Comment:
The generated `KafkaTopic CR''s` is weird and looks like kubebuilder replaces every `'` with `''`, not sure if there is a way to make it not do that for us

Reply:
Weird :) KafkaTopic CR''s ...

Reply:
yes, this has been taken care of with the latest commits